api/swagger.yml

```diff
@@ -672,15 +672,21 @@ components:
       properties:
         id:
           type: string
-          description: a unique identifier for the user.
+          description: A unique identifier for the user.
         creation_date:
           type: integer
           format: int64
           description: Unix Epoch in seconds
         friendly_name:
           type: string
+          description: |
+            A shorter name for the user than the id. Unlike id it does not identify the user (it
+            might not be unique). Used in some places in the UI.
         email:
           type: string
+          description: |
+            The email address of the user. If API authentication is enabled, this field is mandatory and will be invited to login.
```

Comment:
Is it unique? (I assume not, but it pays to be explicit)
Is it case sensitive?

Reply:
@guy-har @Isan-Rivkin same here

Reply:
lakeFS doesn't have such restrictions, in most cases we connect with an external auth provider which does cause the email to be unique